Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -538,13 +579,103 @@ public bool ShowRotationHandleForZ
         [SerializeField]
         [Tooltip("Check to draw a tether point from the handles to the hand when manipulating.")]
         private bool drawTetherWhenManipulating = true;
-
         public bool DrawTetherWhenManipulating
         {
             get { return drawTetherWhenManipulating; }
             set { drawTetherWhenManipulating = value; }
         }
 
+        [Header("Proximity")]
+        [SerializeField]
+        private bool proximityEffectActive = true;
+        public bool ProximityEffectActive
+        {
+            get
+            {
+                return proximityEffectActive;
+            }
+            set
+            {
+                proximityEffectActive = value;
+            }
+        }
+        [SerializeField]
+        [Tooltip("How far away should the hand be from a handle before it starts scaling the handle?")]
+        [Range(0.005f, 0.1f)]
+        private float handleMediumProximity = 0.005f;
+        [SerializeField]
+        [Tooltip("How far away should the hand be from a handle before it activates the close-proximity scaling effect?")]
+        [Range(0.001f, 0.01f)]
+        private float handleCloseProximity = 0.0015f;
+        [SerializeField]
+        [Tooltip("A Proximity-enabled Handle scales by this amount when a hand moves out of range")]
+        private float farScale = 0.0f;//0.01f;
```

Comment:
suggestion: use 1.0f as default value? we don't want users to set this up as 0.0f - so we shouldn't use it as default

Reply:
@thalbern In this update, the handle is invisible(scale = 0) in default. The size increases based on the hand proximity.